advocacy_docs/partner_docs/Cyberark/05-Usingcyberark.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+After you have configured your EDB Postgres Advanced Server and CyberArk, as stated in the Configuring section, you will be able to then use CyberArk (CPM Central Policy Manager) functionality. 
```

Comment:
would say:
you will be able to then use the CyberArk...etc.

Reply:
done